script/release-flynn

```diff
@@ -116,15 +116,16 @@ main() {
     manifest="$(new_release_manifest)"
   fi
 
-  info "checking for existing release"
-  if jq --exit-status ".versions[] | select(.commit==\"${commit}\")" <<< "${manifest}"; then
-    fail "commit ${commit} has already been released"
-  fi
+  info "getting previous release tag"
+  local previous=$(git tag --list v* --sort "v:refname" | tail -n 1)
```

Comment:
You need to quote `v*` to avoid bash expanding it to a list of files:

```
# unquoted
$ bash -x -c 'git tag --list v*' | wc -l
+ git tag --list version.json
       0

# quoted
$ bash -x -c 'git tag --list "v*"' | wc -l
+ git tag --list 'v*'
     203
```